### ISSUE TITLE
Changing the php.ini file has no effect.

### ISSUE THREAD

**Issue by unrl** [bitnami/bitnami-docker-joomla#14]
Hi,

the mounted /php_data/conf directory is not part of the config file path. PhpInfo says:

Configuration File (php.ini) Path:  /opt/bitnami/php/lib
Loaded Configuration File:  (none)
Scan this dir for additional .ini files:    (none)
Additional .ini files parsed:   (none)

So modifying the /php_data/conf/php.ini has no effect.

**Comment by sameersbn** [comment]
To customize the php configuration, you need to mount a directory with your at the `/bitnami/php` path (fe. `--volume /php_data:/bitnami/php`). If the `/php_data` directory is empty, the container will initialize it with the defaults and you should see the configurations available in `/php_data/conf` directory which would include the `php.ini` file as well.

After updating the `php.ini` file, you need to restart the Joomla container for the changes to take effect using:

```bash
docker restart <container-name>
```

or 

```bash
docker-compose restart <service-name>
```

**Comment by unrl** [comment]
The php.ini which is located in /php_data/conf/ will never be loaded. PHP searches for .ini files in /opt/bitnami/php/lib (/opt/bitnami/php/lib is not /bitnami/php)

**Comment by sameersbn** [comment]
The `/opt/bitnami/php/lib/php.ini` file is indirectly symlinked to the file present in `/bitnami/php/conf/php.ini` (see below):

```bash
$ root@494f4fe701cf:~# ls -la /opt/bitnami/php/lib/php.ini 
lrwxrwxrwx 1 root root 29 Sep  2 10:51 /opt/bitnami/php/lib/php.ini -> /opt/bitnami/php/conf/php.ini

$ root@494f4fe701cf:~# ls -la /opt/bitnami/php/conf/php.ini
-rw-r--r-- 1 root root 72732 Sep  2 10:53 /opt/bitnami/php/conf/php.ini

$ root@494f4fe701cf:~# ls -la /opt/bitnami/php/conf        
lrwxrwxrwx 1 root root 17 Sep  2 10:51 /opt/bitnami/php/conf -> /bitnami/php/conf
```

Hope this helps. Thanks

**Comment by unrl** [comment]
I don't know why, but this is what i get:

`root@28222abaa40c:/# ls -la /opt/bitnami/php/lib/php.ini                                                                                                                                                                                                                                
ls: cannot access /opt/bitnami/php/lib/php.ini: No such file or directory

root@28222abaa40c:/# ls -la /opt/bitnami/php/conf/php.ini
-rw-r--r-- 1 root root 72791 Sep  2 12:34 /opt/bitnami/php/conf/php.ini

root@28222abaa40c:/# ls -la /opt/bitnami/php/conf  
lrwxrwxrwx 1 root root 17 Sep  1 18:12 /opt/bitnami/php/conf -> /bitnami/php/conf`

Seems to be a local problem, i'm gonna re-pull everything and test this again. Thank you

**Comment by unrl** [comment]
I figured out that the symlink breaks, after docker-compose recreates the container. (after changing docker-compose.yml)

```
root@0196ccb8cdeb:/# ls -la /opt/bitnami/php/lib/php.ini 
lrwxrwxrwx 1 root root 29 Sep  2 18:40 /opt/bitnami/php/lib/php.ini -> /opt/bitnami/php/conf/php.ini

root@n18 /mnt/docker/test/php_data/conf # docker stop test 
root@n18 /mnt/docker/test/php_data/conf # docker-compose up -d
Recreating test

root@n18 /mnt/docker/test/php_data/conf # docker exec -it test bash
root@aec830469159:/# ls -la /opt/bitnami/php/lib/php.ini 
ls: cannot access /opt/bitnami/php/lib/php.ini: No such file or directory
```

**Comment by sameersbn** [comment]
@unrl Sorry for the long delay. This issue has been resolved in #46 and available in release `bitnami/joomla:3.6.5-r2`
